plugins/executor.js

```diff
@@ -17,6 +17,15 @@ const SCHEMA_STOP = Joi.object().keys({
     annotations: Annotations.annotations,
     buildId
 }).required();
+const SCHEMA_START_PERIODIC = Joi.object().keys({
+    pipeline: Joi.object().required(),
```

Comment:
Should `pipeline` and `job` use schemas from the models? (https://github.com/screwdriver-cd/data-schema/blob/master/models/pipeline.js#L65, https://github.com/screwdriver-cd/data-schema/blob/master/models/job.js#L54)